### PR TITLE
BugFixes

### PR DESCRIPTION
**Pull request by Gayathri-Anamala**
# TITLE

## Description
Summary of changes

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [Jira issue #](fill-in.org)

### CODE REVIEW

File: app/cdap/components/GridTable/components/Breadcrumb/index.tsx [hunk 1]
```diff
@@ -27,10 +27,16 @@ const BreadCrumb = ({ datasetName }) => {
   return (
     <Box className={classes.breadCombContainer}>
       <Breadcrumbs separator={<NavigateNextIcon fontSize="small" />} aria-label="breadcrumb">
-        <Link color="inherit" to={`/ns/${getCurrentNamespace()}/home`}>
+        <Link
+          className={classes.weight400 + ' ' + classes.home}
```
Comment (adityadivami) [inline app/cdap/components/GridTable/components/Breadcrumb/index.tsx]: `${class1} ${class1}`

File: app/cdap/components/GridTable/components/Breadcrumb/index.tsx [hunk 2]
```diff
@@ -27,10 +27,16 @@ const BreadCrumb = ({ datasetName }) => {
...
+          to={`/ns/${getCurrentNamespace()}/home`}
+        >
           Home
         </Link>
-        <Link color="inherit" to={`/ns/${getCurrentNamespace()}/datasources/${`select-dataset`}`}>
+        <Link
+          className={classes.weight400 + ' ' + classes.dataset}
```
Comment (adityadivami) [inline app/cdap/components/GridTable/components/Breadcrumb/index.tsx]: `${class1| ${class1}`

File: app/cdap/components/GridTable/components/Breadcrumb/styles.ts [hunk 3]
```diff
@@ -25,4 +26,18 @@ export const useStyles = makeStyles({
     marginRight: '30px',
     marginLeft: '34px',
   },
+  weight400: {
```
Comment (adityadivami) [inline app/cdap/components/GridTable/components/Breadcrumb/styles.ts]: can u change the classname to more meaningful, it is not just about weight = 400
plz update

**Review by adityadivami — CHANGES_REQUESTED** [review]

File: app/cdap/components/GridTable/components/Breadcrumb/styles.ts [hunk 4]
```diff
@@ -25,4 +26,18 @@ export const useStyles = makeStyles({
     marginRight: '30px',
     marginLeft: '34px',
   },
+  breadcrumblabel: {
```
Comment (adityadivami) [inline app/cdap/components/GridTable/components/Breadcrumb/styles.ts]: this should be camel case
plz maintain across

File: app/cdap/components/GridTable/components/Breadcrumb/index.tsx (hunk 2 above)
Comment (adityadivami) [reply app/cdap/components/GridTable/components/Breadcrumb/index.tsx]: Done

File: app/cdap/components/GridTable/components/Breadcrumb/index.tsx (hunk 1 above)
Comment (Gayathri-Anamala) [reply app/cdap/components/GridTable/components/Breadcrumb/index.tsx]: Done

File: app/cdap/components/GridTable/components/Breadcrumb/styles.ts (hunk 3 above)
Comment (Gayathri-Anamala) [reply app/cdap/components/GridTable/components/Breadcrumb/styles.ts]: Done

File: app/cdap/components/GridTable/components/Breadcrumb/styles.ts (hunk 4 above)
Comment (Gayathri-Anamala) [reply app/cdap/components/GridTable/components/Breadcrumb/styles.ts]: Done

**Pull request closed without merging**